util/zenfs.cc

```diff
@@ -8,6 +8,7 @@
 #include <gflags/gflags.h>
 #include <rocksdb/file_system.h>
 #include <rocksdb/plugin/zenfs/fs/fs_zenfs.h>
+#include <string.h>
```

Comment:
Use c++ headers if possible
```suggestion
#include <cstring>
```

Reply:
Ok.